djautotask/sync.py

```diff
@@ -1694,6 +1694,7 @@ def _assign_field_data(self, instance, object_data):
         instance.active = object_data.get('isActive')
         instance.last_activity_date = object_data.get('lastActivityDate')
         instance.phone = object_data.get('phone')
+        instance.owner_resource = object_data.get('owner_resource')
```

Comment:
`owner_resource` needs be replace with `ownerResourceID`.

Reply:
Actually since it's a FK relationship we need to put it in `related_meta`. I'll make the change.